catalog/spatial/wps/spatial-process-api/src/main/java/org/codice/ddf/spatial/process/api/description/Metadata.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.process.api.description;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/** This class is Experimental and subject to change */
+public class Metadata {
+
+  private List<String> keywords = Collections.emptyList();
```

Comment:
I left it a list because it was easiest but at this point I don't really know how we would use it, the examples in the OGC spec are all really small lists too. Discoverability isn't really part of the WPS spec so I think in the end full list of keywords would be retrieved to be index elsewhere anyway.